packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -30,116 +31,102 @@ import { AccountsItemLoader } from 'App/Components/Layout/Header/Components/Prel
 import AccountList from './account-switcher-account-list.jsx';
 import AccountWrapper from './account-switcher-account-wrapper.jsx';
 
-class AccountSwitcher extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            active_tab_index: !props.is_virtual || props.should_show_real_accounts_list ? 0 : 1,
-            is_demo_deriv_visible: true,
-            is_demo_dmt5_visible: true,
-            is_real_deriv_visible: true,
-            is_real_dmt5_visible: true,
-            has_autohide: true,
-        };
-    }
-
-    toggleVisibility = section => {
-        this.setState({ [`is_${section}_visible`]: !this.state[`is_${section}_visible`] });
-    };
-
-    setWrapperRef = node => {
-        this.wrapper_ref = node;
-    };
-
-    updateAccountTabIndex = index => {
-        this.setState({ active_tab_index: index });
-    };
-
-    handleClickOutside = event => {
-        const accounts_toggle_btn = !event.target.classList.contains('acc-info');
-        if (
-            this.wrapper_ref &&
-            !this.wrapper_ref.contains(event.target) &&
-            this.props.is_visible &&
-            accounts_toggle_btn
-        ) {
-            this.closeAccountsDialog();
+const AccountSwitcher = props => {
+    const [active_tab_index, setActiveTabIndex] = React.useState(
+        !props.is_virtual || props.should_show_real_accounts_list ? 0 : 1
+    );
+    const [is_demo_deriv_visible, setDemoDerivVisible] = React.useState(true);
+    const [is_demo_dmt5_visible, setDemoDmt5Visible] = React.useState(true);
+    const [is_real_deriv_visible, setRealDerivVisible] = React.useState(true);
+    const [is_real_dmt5_visible, setRealDmt5Visible] = React.useState(true);
```

Comment:
Isn't it better to name variables like `is_deriv_demo_visible`? That's exactly how we speak and write.